File: server/src/main/java/com/fott/use_mybatis/domain/OTT.java
package com.fott.use_mybatis.domain;

import com.sun.istack.internal.NotNull;
import lombok.*;

@NoArgsConstructor
@RequiredArgsConstructor
@Getter
@Setter
@ToString
public class OTT {
    private Long ottId;

    @NotNull
    private String ottName;
    @NotNull
    private Integer maxUser;
    @NotNull
    private Integer price;
}

File: front/src/models/ott.ts
import React from "react";

export interface Ott {
    ottId : number;
    maxUser: number|null;
    price: number|null;
    ottName: string|null;
}

export const defaultData : Ott ={
    price : null,
    maxUser: null,
    ottId: -1,
    ottName: ""
    
}